pipes/WDL/tasks/tasks_16S_amplicon.wdl

```diff
@@ -80,9 +81,11 @@ task trim_reads {
         #Boolean not_default = false
         String  forward_adapter         = "CTGCTGCCTCCCGTAGGAGT"
         String  reverse_adapter         = "AGAGTTTGATCCTGGCTCAG"
-        Int?    min_length              = 1
-        Boolean? keep_untrimmed_reads   = false
-
+        Int    min_length              = 1
```

Comment:
Why did you make this variable required--it's used in a optional manner in the command block down below

Reply:
So when i passed this onto miniwdl, it kept saying that I did not need to add "?" to make it optional so i removed it

Reply:
So when i passed this onto miniwdl, it kept saying that I did not need to add "?" to make it optional so i removed it